src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -216,37 +216,37 @@ public partial class UpdateDurableFunction
 
                 try
                 {
-                    instanceMetadataList = await context.CallActivityWithRetryAsync<IReadOnlyList<InstanceMetadata>>(
-                        nameof(UpdateInstanceBlobsV2Async),
+                    UpdateInstanceResponse response = await context.CallActivityWithRetryAsync<UpdateInstanceResponse>(
+                        nameof(UpdateInstanceBlobsV3Async),
                         _options.RetryOptions,
                         new UpdateInstanceBlobArgumentsV2(input.Partition, instances, input.ChangeDataset));
 
-                    await context.CallActivityWithRetryAsync(
-                        nameof(CompleteUpdateStudyV2Async),
+                    instanceMetadataList = response.InstanceMetadataList;
+
+                    if (response.Errors != null && response.Errors.Any())
```

Comment:
Don't use `Any()` when we have `Count`:
```suggestion
                    if (response.Errors?.Count > 0)
```